lib/click-handler.js

```diff
@@ -110,19 +90,38 @@ function onClick(event) {
     return;
   }
 
-  tryLoad(urls, (err, url, res) => {
-    if (err) {
-      showTooltip($tooltipTarget, SORRY);
+  const track = action =>
+    chrome.runtime.sendMessage({
+      type: 'track',
+      payload: {
+        category: 'fetch',
+        action,
+      },
+    });
 
-      return console.error(err); // eslint-disable-line no-console
-    }
+  try {
+    // TODO remove this and fix tests
+    // Maybe use the following for mocking fetch?
+    // https://github.com/speedskater/babel-plugin-rewire
+    chrome.runtime.sendMessage({
+      type: 'fetch',
+      payload: urls,
+    });
```

Comment:
In this case I would remove the code from here (since it isn't used anymore) and disable the related test until we find a suitable solution